src/main/java/de/retest/recheck/TestCaseFinder.java

```diff
@@ -51,6 +53,9 @@
 			"org.junit.jupiter.api.RepeatedTest", //
 			"org.junit.jupiter.params.ParameterizedTest" ) );
 
+	private static final List<String> suffixes = Arrays.asList( "Test", "IT", "TestCase", "ITCase", "Tests" );
```

Comment:
"suffixes" and "prefixes" of what? 🤷‍♂️ Just like with `testCaseAnnotations`, it may become more clear if you do something like:

```java
private static final List<String> testClassSuffixes = Arrays.asList( //
        // Maven Surefire
        "Test",
        // ...
        // Maven Failsaife
        "IT"
         // ...
```

I suggest to do the same for the prefixes.